github-for-jira.sd.yml

```diff
@@ -276,9 +276,6 @@ environmentOverrides:
           default_vanity_dns: false
           domain:
             - github.stg.atlassian.com
-          upstream_address:
-            - address: github-for-jira.stg.services.atlassian.com
-          rewrite: github-for-jira.stg.services.atlassian.com
```

Comment:
this is not needed anymore, because by default, global edge will route directly to the service that owns this descriptor